src/demos/head.js

```diff
@@ -1,32 +1,58 @@
 (function() {
-  const root = window.location.pathname.split("demos").shift();
+  const CSS = ["demos/demos.css", "build/calcite-app.css", "vendor/@esri/calcite-components/calcite.css"];
+
+  const SCRIPTS = [
+    {
+      src: "build/calcite-app.esm.js",
+      type: "module"
+    },
+    {
+      src: "build/calcite-app.js",
+      noModule: true
+    },
+    {
+      src: "vendor/@esri/calcite-components/calcite.esm.js",
+      type: "module"
+    },
+    {
+      src: "vendor/@esri/calcite-components/calcite.js",
+      noModule: true
+    }
+  ];
+
+  if (location.port >= 3333 && location.port < 3340) {
+    SCRIPTS.push({
+      src: "demos/demoPageReloader.js"
+    });
+  }
+
+  const IS_IE = /*@cc_on!@*/ false || !!document.documentMode; // Internet Explorer 6-11
```

Comment:
Can the user agent be used instead or why is `@cc_on!@` needed?

Nitpick: can the var be `IS_IE11` or why do we need to focus on older versions?